projekt_do_pracy_na_labach/fiszki_backend/compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+services:
+  frontend:
+    image: fiszki_app:v1
+    ports:
+      - 80:80
+    depends_on:
+      - backend
+    networks:
+      - app-network
+
+  backend:
+    image: fiszki_backend:v1
+    ports: 
+      - 5000:5000
```

Comment:
Tu jak robiłem remergea to nie zauważyłem a powinno być 8000, bo taki port ustawiliśmy w pliku